### DISCUSSION TITLE
graphql's response has repeated data (services or endpoints)

### DISCUSSION THREAD

**Discussion by AngryMills** [apache/skywalking#7481] [Q&A] [answered]
Please answer these questions before submitting your issue.

- Why do you submit this issue?
- [√] Question or discussion
- [√] Bug

___
### Question
- What do you want to know?
uri: /graphql
parameters: {"query":"query queryServices($duration: Duration!,$keyword: String!) {\n    services: getAllServices(duration: $duration, group: $keyword) {\n      key: id\n      label: name\n      group\n    }\n  }","variables":{"duration":{"start":"2021-08-18 1359","end":"2021-08-18 1429","step":"MINUTE"},"keyword":""}}
response:
{"data":{"services":[{"key":"YXJjaC1hcHAtZmxvdw==.1","label":"arch-app-flow","group":""},{"key":"YXJjaC1hcHAtZmxvdw==.1","label":"arch-app-flow","group":""}]}}
repeated services have same keys and labels.
![image](https://user-images.githubusercontent.com/19707438/129856190-17672707-eb2f-41ea-b5f9-e4455f1a092d.png)

### Bug
- Which version of SkyWalking, OS, and JRE?
skywalking 8.6.0
centos 7
jdk 8

- What happened?
I don't know how this happened，maybe add some code to remove duplicate data in query-graphql-plugin?
I find that, graphql  of query queryEndpoints has the same problem.
![image](https://user-images.githubusercontent.com/19707438/129857045-6e735051-c75d-463f-ae0d-4febc0b9728a.png)

___
Thanks for answering.

**Comment by wu-sheng** [comment, marked as answer]
This is mostly caused by your storage(ES Cluster performance issue). In a period of time, it is not. readable, so, OAP insert the second record into the index. It is harmless. We may consider remove duplicate in the 8.8.0.

**Reply by wu-sheng** [reply]
I added a patch https://github.com/apache/skywalking/pull/7482 to make users less confused in UI. But the perf issue still exists.

**Reply by AngryMills** [reply]
Copy that , thanks.

**Reply by git1e** [reply]
My Skywalking version is 8.7.0 ，jdk8 ,centos7.  and  I have the same problem，how did you solve it？
